README.md

```diff
@@ -32,39 +32,169 @@
 - For changes, see the [Changelog](CHANGELOG.md)
 - We are always welcoming contributions. For the development notes: [Contributing](CONTRIBUTING.md)
 
+## Installation and Running: 
 
-### To install from PyPI:
+### **Method 1: Install via PyPI and Run with Docker Compose or Podman Compose**
 
-```shell
-pip install stac_fastapi.elasticsearch
-```
-or
-```
-pip install stac_fastapi.opensearch
-```
+#### **Prerequisites**
 
-## Build Elasticsearch API backend
+- [**Docker Compose**](https://docs.docker.com/compose/install/) or [**Podman Compose**](https://podman-desktop.io/docs/compose) installed and running on your machine. In all the follwoing steps instead of `docker-compose` you can use `podman-compose` as well.
 
-```shell
-docker-compose up elasticsearch
+#### **Step 1: Install STAC-FastAPI**
+
+To install STAC-FastAPI with Elasticsearch or OpenSearch backend support, execute the following commands:
+
+- **For Elasticsearch backend:**
+
+  ```bash
+  pip install stac_fastapi.elasticsearch
+  ```
+
+- **For OpenSearch backend:**
+
+  ```bash
+  pip install stac_fastapi.opensearch
+  ```
+
```

Comment:
This isn't too clear in the present version of the readme but the pip install step is not a step that's needed to run the docker compose steps as listed down below. You could:
1. docker compose up elasticsearch
2. pip install
3. run the app
or 
1. docker compose up elasticsearch
2. docker compose build app-elasticsearch
3 docker compose up app-elasticsearch

This should probably be made clear in the readme that these options are for developing the api locally.

Reply:
via the latest changes, I think it covers the production option as well

Reply:
I agree with Jon here. 

The pip commands should be in a separate section: `Installation`

To develop the API locally the only command needed is 
```
docker compose up app-opensearch
```
or
```
docker compose up app-elasticsearch
```